agent/api/task/task.go

```diff
@@ -1206,6 +1209,13 @@ func (task *Task) addNetworkResourceProvisioningDependency(cfg *config.Config) e
 		container.BuildContainerDependency(NetworkPauseContainerName, apicontainerstatus.ContainerResourcesProvisioned, apicontainerstatus.ContainerPulled)
 		pauseContainer.BuildContainerDependency(container.Name, apicontainerstatus.ContainerStopped, apicontainerstatus.ContainerStopped)
 	}
+
```

Comment:
Do we need the dependency for pause container to make sure task resource is cleaned beforehand?

Reply:
i removed it because i found that we can't remove efs volume before pause container stops, because docker doesn't allow removing a volume when a container using it hasn't been removed (even if container has stopped), and we don't remove container until all containers are stopped